### ISSUE TITLE
how to use the nndmd in  other nonlinear dynamic system

### ISSUE THREAD

**Issue by fdu-goya** [dynamicslab/pykoopman#60]
In addition, for other complex nonlinear systems, using the NNDMD in the code library for Koopman learning does not seem to work well. Could you provide more examples of other nonlinear cases?

**Comment by fdu-goya** [comment]
Because the results from using nndmd on other nonlinear dynamical systems are particularly poor, and nndmd is mainly based on LBFGS optimization, which is a second-order algorithm, its computational speed is particularly slow when the model requires a large number of epochs to run.

**Comment by fdu-goya** [comment]
When I applied the NNDMD to the Duffing oscillator system, I adjusted the depth of the neural network, the number of neurons, activation functions, and optimizers multiple times but still did not achieve a good approximation of the Koopman operator. I would like to know how to construct a better neural network for this NNDMD to obtain a better approximation of the Koopman operator for the Duffing oscillator. I would greatly appreciate it if you could provide more examples of nonlinear dynamical systems.

**Comment by fdu-goya** [comment]
dlk_regressor = pk.regression.NNDMD(mode="Dissipative",dt=unforcing_sys.dt, look_forward=look_forward,
                                    config_encoder=dict(input_size=2,
                                                        hidden_sizes=[64]*3,
                                                        output_size=10,
                                                        activations="elu"),
                                    config_decoder=dict(input_size=10, hidden_sizes=[],
                                                        output_size=2, activations="linear"),
                                    batch_size=512, lbfgs=False,
                                    normalize=True, normalize_mode='equal',
                                    normalize_std_factor=1.0,
                                    trainer_kwargs=dict(max_epochs=2000,accelerator="gpu", devices=1))

hello,These are the parameters I set for the Duffing oscillator, and the training loss obtained is as follows. The results are not very good, and I am unable to learn the correct Koopman operator.

**Comment by pswpswpsw** [comment]
can you give me a zip file that contains the data and the jupoyter notebok ?
